### PR TITLE
Shrestha/Added python function to query ngraph-bridge library

### PR DESCRIPTION
**Pull request by shresthamalik**
Add function to query ngraph bridge python module for build flag --enable_variables_and_optimizers

### CODE REVIEW

File: python/ngraph_bridge/__init__.in.py [hunk 1]
```diff
@@ -39,7 +39,7 @@
     'enable', 'disable', 'is_enabled', 'backends_len', 'list_backends',
     'set_backend', 'is_supported_backend', 'get_currently_set_backend_name',
     'start_logging_placement', 'stop_logging_placement',
-    'is_logging_placement', '__version__'
+    'is_logging_placement', '__version__', 'is_grappler_enabled' ,'are_variables_enabled'
```
Comment (avijit-nervana) [inline python/ngraph_bridge/__init__.in.py]: Place this to the next line

File: python/ngraph_bridge/__init__.in.py [hunk 2]
```diff
@@ -188,10 +189,15 @@ def is_logging_placement():
 def is_grappler_enabled():
     return ngraph_bridge_lib.ngraph_tf_is_grappler_enabled()
 
+def are_variables_enabled():
```
Comment (avijit-nervana) [inline python/ngraph_bridge/__init__.in.py]: Also add a function to return the CXX11ABI (i.e., a python function that returns an int reflecting the ngraph_bridge_lib.ngraph_tf_cxx11_abi_flag)
Something like: 
```
int cxx11_abi_flag() { return ngraph_bridge_lib.ngraph_tf_cxx11_abi_flag }
```

Comment (shresthamalik) [reply python/ngraph_bridge/__init__.in.py]: It's already there, no?
https://github.com/NervanaSystems/ngraph-tf/blob/master/python/ngraph_bridge/__init__.in.py#L195

Comment (shresthamalik) [reply python/ngraph_bridge/__init__.in.py]: oh, python function is missing. Working on it.

**Pull request merged**